ccpa_cmplibrary/src/main/java/com/sourcepoint/ccpa_cmplibrary/CCPAConsentLib.java

```diff
@@ -411,10 +412,14 @@ void storeData(){
         storeClient.setMetaData(metaData);
     }
 
+    private boolean hasParent(View v) {
```

Comment:
What if we call this method `isViewPresented`? This way we indicate what are we using it for.

Reply:
👍